curvefs/proto/topology.proto

```diff
@@ -381,14 +381,28 @@ message CommitTxResponse {
 }
 
 message GetCopysetInfoRequest {
-    required uint32 copysetId = 1;
+    required uint32 poolId = 1;
+    required uint32 copysetId = 2;
+}
+
+message CopysetInfo {
+    required uint32 poolId = 1;
+    required uint32 copysetId = 2;
+    repeated common.Peer peers = 3;
 }
 
 message GetCopysetInfoResponse {
     required TopoStatusCode statusCode = 1;
-    required curvefs.mds.heartbeat.CopySetInfo copysetInfo = 2;
+    optional CopysetInfo copysetInfo = 2;
+}
+
+message GetCopysetsInfoRequest {
+    repeated GetCopysetInfoRequest copysets= 1;
 }
 
+message GetCopysetsInfoResponse {
+    repeated GetCopysetInfoResponse copysetsInfo = 1;
+}
```

Comment:
The interfaces 'GetCopyset' and 'GetCopysets' are redundant.
Maybe can change the interface 'GetCopyset':
message CopysetKey{
    required uint32 poolId = 1;
    required uint32 copysetId = 2;
}
message GetCopysetInfoRequest {
    repeated CopysetKey key = 1;
}
message GetCopysetInfoResponse {
    repeated curvefs.mds.heartbeat.CopySetInfo copysetInfo;
}

Reply:
> The interfaces 'GetCopyset' and 'GetCopysets' are redundant. Maybe can change the interface 'GetCopyset': message CopysetKey{ required uint32 poolId = 1; required uint32 copysetId = 2; } message GetCopysetInfoRequest { repeated CopysetKey key = 1; } message GetCopysetInfoResponse { repeated curvefs.mds.heartbeat.CopySetInfo copysetInfo; }

this can fix in next version.